src/chrome-launcher.ts

```diff
@@ -181,6 +182,24 @@ class Launcher {
     return chromeFinder[getPlatform() as SupportedPlatforms]();
   }
 
+  static getFirstInstallation(): string[] {
```

Comment:
I think this actually makes sense to live right next to getInstallations so we can keep the priorities in sync

maybe also limit the short-circuit to `os.platform() === 'darwin'`?